Doc/library/sqlite3.rst

```diff
@@ -343,17 +343,17 @@ Module functions
    .. audit-event:: sqlite3.connect database sqlite3.connect
    .. audit-event:: sqlite3.connect/handle connection_handle sqlite3.connect
 
-   .. versionadded:: 3.4
-      The *uri* parameter.
+   .. verisonchanged:: 3.4
```

Comment:
```suggestion
   .. versionchanged:: 3.4
```

Fix typo in directive name (causing docs build to fail)

Reply:
Thanks!